### DISCUSSION TITLE
Broken links in domains-blocklist.conf

### DISCUSSION THREAD

**Discussion by remyabel2** [DNSCrypt/dnscrypt-proxy#2009] [Potential issues]
I know the config provided is just an example, but some of the links need to be revisited. For example:

- `https://raw.githubusercontent.com/PolishFiltersTeam/KADhosts/master/KADhosts_without_controversies.txt` is blank (only has comments)
- `https://gitlab.com/CHEF-KOCH/cks-filterlist/-/raw/master/Anti-Corp/Spotify/Spotify-HOSTS.txt ` is a broken link because CHEF-KOCH was suspended for stealing people's work
- `https://www.gameindustry.eu/files/hosts.txt` 404's as well

**Comment by jedisct1** [comment]
Fixed in e6089449b6b7be93aca2c045fe2fcf9f7f9fa4bc thanks!

**Comment by ghost** [comment]
GameIndustry only changed the link, so it can be re-added:

- `https://hosts.gameindustry.eu/pc-main-template/dl/`
- `https://raw.githubusercontent.com/KodoPengin/GameIndustry-hosts-Template/master/Main-Template/hosts`

Edit: See PR #2014.
